presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -146,6 +146,14 @@ class TaskManager {
   // in exec/Task.h).
   std::array<size_t, 5> getTaskNumbers(size_t& numTasks) const;
 
+  /// Populate the blocked tasks (failing to take lock on the mutex), and long
```

Comment:
```
s/Populate/Populates/
```
How about
```
/// Invoked to check the long running operation calls in the system. If the function fails to get ... from a task due to the lock timeout, it adds the task to 'blockedTasks'. Otherwise, it adds all ... to 'longRunningOpCalls'. The function returns false ...
```